src/lib/signature.ts

```diff
@@ -235,21 +236,23 @@ class Signature extends CircuitValue {
    * Signs a message using a {@link PrivateKey}.
    * @returns a {@link Signature}
    */
-  static create(privKey: PrivateKey, msg: Field[]): Signature {
+  static create(privKey: PrivateKey, msg: Field[], networkId: NetworkId = 'testnet'): Signature {
```

Comment:
revert - `Signature.create()` is the o1js version of `signFields`, and is unrelated to the Mina network